test/integration/targets/iam_policy/aliases

```diff
@@ -1,2 +1,2 @@
 cloud/aws
-posix/ci/cloud/group4/aws
+unsupported
```

Comment:
Since there isn't a way to stipulate the contents of the inline policies that may be attached I marked these as unsupported. Sorry for the delay in getting to these! Thanks for spending time on it.